ci/browser_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+SAUCECONNECT_URL="http://saucelabs.com/downloads/sc-4.3.16-linux.tar.gz"
+SAUCECONNECT_USERNAME="chriseth"
+SAUCECONNECT_ACCESSKEY="b781828a-9e9c-43d8-89d4-2fbb879595ca"
+SAUCECONNECT_JOBIDENTIFIER="browsersolidity_tests_${TRAVIS_JOB_NUMBER}"
+SAUCECONNECT_READYFILE="sc.ready"
+TEST_EXITCODE=0
+
+npm run build
+npm run serve &
+
+wget $SAUCECONNECT_URL
+tar -zxvf sc-4.3.16-linux.tar.gz
+./sc-4.3.16-linux/bin/sc -u $SAUCECONNECT_USERNAME -k $SAUCECONNECT_ACCESSKEY -i $SAUCECONNECT_JOBIDENTIFIER --readyfile $SAUCECONNECT_READYFILE &
+while [ ! -f $SAUCECONNECT_READYFILE ]; do
+  sleep .5
+done
+
+function updateTestExitCode() {
+  if [ $? -eq 1 ]
+  then
+    TEST_EXITCODE=1
+  fi
+}
+
+npm run browser-test-remote-firefox
```

Comment:
If you add `set -e` above, this of course does not work anymore :-)

But it can be replaced by: `npm run browser-test-remote-firefox || TEST_EXITCODE=1`